notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>If you are running this notebook locally, <s>you will need to</s> install the&nbsp;<a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/raw/02157e0f7cabc613231f7ef82be00f099eaf4805/notebooks/official/automl/(https://cloud.google.com/sdk)" target="_blank">Cloud SDK</a>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done</p>